topdown/strings.go

```diff
@@ -87,19 +87,19 @@ func builtinConcat(a, b ast.Value) (ast.Value, error) {
 	return ast.String(strings.Join(strs, string(join))), nil
 }
 
-func builtinIndexOf(a, b ast.Value) (ast.Value, error) {
-	runesEqual := func(a, b []rune) bool {
-		if len(a) != len(b) {
+var runesEqual = func(a, b []rune) bool {
```

Comment:
This strikes me as a bit uncommon. Why not 
```suggestion
func runesEqual(a, b []rune) bool {
```